src/main/java/com/epam/flightscontroltool/boundary/endpoint/AirplaneEndpoint.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.flightscontroltool.boundary.endpoint;
+
+import com.epam.flightscontroltool.boundary.dto.AirplaneDto;
+import com.epam.flightscontroltool.boundary.request.AirplaneRequest;
+import com.epam.flightscontroltool.control.service.AirplaneService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/airplanes")
+public class AirplaneEndpoint {
+
+    private final AirplaneService airplaneService;
+
+    @ApiOperation(value = "Retrieving a page of airplanes")
+    @ApiResponses({
+            @ApiResponse(code = 200, message = "A page of airplanes"),
+    })
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    public Page<AirplaneDto> getAirplanesPage(
+            @ApiParam("Company id") @RequestParam(value = "companyId", required = false) Long companyId,
+            @ApiParam("Page number") @RequestParam(value = "page", required = false, defaultValue = "0") Integer page,
```

Comment:
You can remove this two request parameters (page and size). And replace it with just Pageable. Spring is pretty intelligent and he is gonna await 2 request parameters ("page" and "size"). Also there are some ways to specify default Pageable - https://stackoverflow.com/questions/27032433/set-default-page-size-for-jpa-pageable-object. I haven't tried all of them.